Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.60.3', '2.73.1'])
+buildPlugin(jenkinsVersions: [null, '2.73.1'])
```

Comment:
Better to just use 2.73.x as the baseline. No one cares about people using dailies prior to a published LTS line.